### PR TITLE
fix(OpStackAdapter): Filter out available l1 tokens when computing outstanding transfers

### PR DESCRIPTION
**Pull request by nicholaspai**
We forgot to include this filter line that we have in other adapters.

### CODE REVIEW

File: src/clients/bridges/op-stack/OpStackAdapter.ts [hunk 1]
```diff
@@ -74,6 +74,7 @@ export class OpStackAdapter extends BaseAdapter {
 
   async getOutstandingCrossChainTransfers(l1Tokens: string[]): Promise<OutstandingTransfers> {
     const { l1SearchConfig, l2SearchConfig } = this.getUpdatedSearchConfigs();
+    const availableL1Tokens = l1Tokens.filter(this.isSupportedToken.bind(this));
```
Comment (james-a-morris) [inline src/clients/bridges/op-stack/OpStackAdapter.ts:77]: Do we need the bind?

Comment (nicholaspai) [reply src/clients/bridges/op-stack/OpStackAdapter.ts:77]: i don't think so but i think good to stay consistent

Comment (james-a-morris) [reply src/clients/bridges/op-stack/OpStackAdapter.ts:77]: Right but `bind` changes the object scope

**Review by james-a-morris — APPROVED** [review]
+1 as long as `bind` isn't an issue

**Pull request merged**